Dockerfile

```diff
@@ -0,0 +1,29 @@
+## Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.## 
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+##     http://www.apache.org/licenses/LICENSE-2.0 
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## 
+
+FROM golang:1.21
+
+WORKDIR /src
+COPY . .
+
+RUN make build
+RUN install -m 755 /src/bin/holodeck /usr/local/bin/holodeck && \
+    install -m 755 /src/scripts/run.sh /usr/local/bin/run.sh && \
+    install -m 755 /src/scripts/cleanup.sh /usr/local/bin/cleanup.sh
+
+RUN echo "nobody:x:65534:65534:Nobody:/:" > /etc_passwd
```

Comment:
Thoughts on using distroless as the final base of the image?

Reply:
I was even thinking on `nvidia/cuda:base` just to align with all the projects.

Reply:
I think the point of a distroless image is to reduce the image size. I would avoid a cuda image if possible since this is another axis to update.

Reply:
yeah, that's why I didn't went that route.